src/main/java/co/cask/hydrator/plugin/batchSource/KafkaBatchSource.java

```diff
@@ -304,7 +304,10 @@ public Schema getMessageSchema() {
     }
 
     public void validate() {
-      getBrokerMap();
```

Comment:
Here is my concern: Although offsets are well known in Kafka, an offset table is very much a CDAP implementation detail. To a user configuring a Kafka source, it seems to me that the only required properties should be Kafka properties, not CDAP properties, no? We can always make a best effort to choose a unique name - say pipeline name + topic name. Only if it fails, or only for advanced users who know what they are doing, we can expose a property to override that default.